### ISSUE TITLE
How to set Learning Rate with Keras

### ISSUE THREAD

**Issue by xqchen** [xqchen/blogs#6]
import keras.backend as K
K.set_value(sgd.lr, 0.5 * K.get_value(sgd.lr))
